Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -2011,7 +2022,7 @@ driverLocationTracking push action driverArrivedAction updateState duration trac
                                                     specialLocationConfig "" sourceSpecialTagIcon
                                                   else
                                                     specialLocationConfig "" destSpecialTagIcon
-                        liftFlow $ updateRoute newPoints markers.destMarker (metersToKm locationResp.distance state) markers.srcMarker specialLocationTag
+                        liftFlow $ updateRoute (updateRouteConfig newPoints markers.destMarker (metersToKm locationResp.distance state) markers.srcMarker specialLocationTag zoomLevel)
```

Comment:
no needed to create separate function. do like this
`startLottieProcess lottieAnimationConfig {rawJson = "record_audio_animation.json", lottieId = (getNewIDWithTag "recordAnimation"), scaleType = "FIT_CENTER", speed = 1.0 }`